Add iOS 12.6 Release Notes (#122)
owncloud/docs-main: modules/ROOT/pages/ios_release_notes.adoc

toc::[]

== iOS App 12.6.0

[discrete]
=== General

This release contains changes and bugfixes. +
Refer to the full change log for a list of bug fixes and changes at {ios-releases-url}/v12.6.0[GitHub, window=_blank].

[discrete]
=== Notable Changes

* Adds a row of recently picked locations to the Location Picker for quick access.

> * Starting this version iOS ownCloud app can be used with Kiteworks PDN version 9.1.1 and higher.

Author: ```suggestion
*This version of the iOS ownCloud app is compatible with the Kiteworks PDN version 9.1.1 and higher.
```

> *This version of the iOS ownCloud app is compatible with the Kiteworks PDN version 9.1.1 and higher.

Author: ```suggestion
* This version of the iOS ownCloud app is compatible with the Kiteworks PDN version 9.1.1 and higher.
```
